app/views/tag/_replication.html.erb

```diff
@@ -2,7 +2,10 @@
 <% if @node.has_power_tag(response_type) %>
 <p class="guided-tags alert alert-info">
   <b>This is marked this as an <%= response_type %> for others to try.</b> <br />
-  <a href="/post?tags=<%= response_type %>:<%= @node.id %><% if response_type == 'response' %>,hidden:response<% end %><%= ',' + (@tagnames.uniq.delete_if{|x| x.match(":") }).join(',') if @tagnames && @tagnames.length > 0%>" class="btn btn-sm btn-default">Try it now</a> &nbsp;
+  <a <% if current_user %> href="/post?tags=<%= response_type %>:<%= @node.id %><% if response_type == 'response' %>,hidden:response<% end %><%= ',' + (@tagnames.uniq.delete_if{|x| x.match(":") }).join(',') if @tagnames && @tagnames.length > 0%>" class="btn btn-sm btn-default">Try it now
+<% else %> <a data-toggle="modal" data-target="#loginModal" class="btn btn-sm btn-default">Try it now </button> </a>  <% end %>
+
+             </a> &nbsp; <br>
```

Comment:
@oorjitchowdhary Please try just adding the `requireLogin` class to the anchor tag. Change `class="btn btn-sm btn-default"` to `class="btn btn-sm btn-default requireLogin"`. Hopefully it will work 😄 . There should be no need to check for current user. Also please remove the changes to `app/views/user_sessions/_new.html.erb`

Reply:
Merry Christmas!!!
Thanks for helping him out. I was busy with other PR review so can't help him much. 
Thanks a lot.